modules/conflict_resolver/php/conflict_resolver.class.inc

```diff
@@ -340,49 +310,57 @@ class Conflict_Resolver extends \NDB_Menu_Filter_Form
         if ($user->hasPermission('access_all_profiles')) {
             // get the list of study sites - to be replaced by the Site object
             $sites = \Utility::getSiteList();
+            foreach ($sites as $key => $site) {
+                unset($sites[$key]);
+                $sites[$site] = $site;
+            }
             if (is_array($sites)) {
                 $sites = array('' => 'All') + $sites;
             }
         } else {
             // allow only to view own site data
             $sites = $user->getStudySites();
+            foreach ($sites as $key => $site) {
+                unset($sites[$key]);
+                $sites[$site] = $site;
+            }
             $sites = array('' => 'All User Sites') + $sites;
         }
 
         // Add form elements
-        $this->addSelect('site', 'Site:', $sites);
-        $this->addSelect('Instrument', 'Instrument:', $instruments);
-        $this->addSelect('visit', 'Visit label:', $visits);
+        $this->addSelect('site', 'For Site:', $sites);
+        $this->addSelect('instrument', 'Instrument:', $instruments);
+        $this->addSelect('visitLabel', 'Visit label:', $visits);
         $this->addBasicText(
-            'CandID',
-            'DCCID:',
+            'candID',
+            'CandID:',
             array(
-             "size"      => 9,
-             "maxlength" => 6,
+             'size'      => 9,
+             'maxlength' => 6,
             )
         );
         $this->addBasicText(
-            'PSCID',
+            'pSCID',
             'PSCID:',
             array(
-             "size"      => 9,
-             "maxlength" => 7,
+             'size'      => 9,
+             'maxlength' => 7,
             )
         );
-        $this->addBasicText('Question', 'Search keyword:');
+        $this->addBasicText('question', 'Question:');
 
         // Project list, if applicable
-        if ($config->getSetting("useProjects")==="true") {
+        if ($config->getSetting('useProjects')==='true') {
             $list_of_projects = array(null => 'All');
             $projectList      = \Utility::getProjectList();
             foreach ($projectList as $key => $value) {
```

Comment:
In order for the Project filter to work, you need to make sure the key for is the same as the value in the $list_of_projects array.